### PR TITLE
Library guidance: clarified intent of the section

### PR DESCRIPTION
**Pull request by pkulikov**
This PR makes the purpose of the following section clear:
https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/#recommendations

This section is not for recommendations, but introduce the types of recommendations used throughout the guidance.

/cc: @JamesNK

### CODE REVIEW

**Review by JamesNK — COMMENTED** [review]
Good improvement. Couple of minor language queries

**Comment by pkulikov** [comment]
@JamesNK thanks for review. Also it was a good test of the "suggestions" feature.

**Review by JamesNK — APPROVED** [review]
Looks good to me. Thanks. A docs review will double check the language before merge.

File: docs/standard/library-guidance/index.md [hunk 1]
```diff
@@ -21,23 +21,23 @@ Aspects of high-quality open-source .NET libraries:
 > [!div class="nextstepaction"]
 > [Get Started](./get-started.md)
 
-## Recommendations
+## Types of recommendations
 
-With each article, there is a list of recommendations for your .NET library using **Do**, **Consider**, **Avoid**, and **Do not**. The wording of each recommendation indicates how strongly it should be followed.
+Articles in this guidance present recommendations of four types: **Do**, **Consider**, **Avoid**, and **Do not**. The type of recommendation indicates how strongly it should be followed.
```
Comment (rpetrusha) [inline docs/standard/library-guidance/index.md]: "Each article presents four types of recommendations" is clearer.

File: docs/standard/library-guidance/index.md [hunk 2]
```diff
@@ -21,23 +21,23 @@ Aspects of high-quality open-source .NET libraries:
...
+You should almost always follow a **Do** recommendation. For example:
 
 **✔️ DO** distribute your library using a NuGet package.
 
 On the other hand, **Consider** recommendations should generally be followed, but there are legitimate exceptions to the rule and you shouldn't feel bad about not following the guidance:
 
 **✔️ CONSIDER** using [SemVer 2.0.0](https://semver.org/) to version your NuGet package.
 
-**Avoid** recommendations are things that are generally not a good idea, but breaking the rule sometimes makes sense:
+**Avoid** recommendations mention things that are generally not a good idea, but breaking the rule sometimes makes sense:
```
Comment (rpetrusha) [inline docs/standard/library-guidance/index.md]: Very good change!! I didn't like "are things" previously, but couldn't think of a better replacement.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This is much clearer, @pkulikov. Thanks for doing it. I've left one suggestion for you to consider.

**Comment by pkulikov** [comment]
@rpetrusha thanks for review; I've addressed your feedback.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @pkulikov. This topic now looks really good. I'll merge your PR.

**Pull request merged**